lib/twofishes/result_fields.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_support/core_ext/string/inflections'
+
+module Twofishes
+  class ResultFields
+    def initialize(hash)
+      @hash = hash
+    end
+
+    def to_h
+      @hash.dup
+    end
+
+    alias_method :to_hash, :to_h
+
+    def [](key)
+      wrap_value @hash[key.to_s.camelize(:lower)]
```

Comment:
Again, stop looking at this, look at #4 :smile: Or have the day off, like the doctor ordered